addons/web/static/src/views/fields/properties/property_value.js

```diff
@@ -0,0 +1,290 @@
+/** @odoo-module **/
+
+// Represent one property value
+// Supports many types and instantiates the appropriate component for it
+
+import { _lt } from "@web/core/l10n/translation";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { CheckBox } from "@web/core/checkbox/checkbox";
+import { DateTimePicker, DatePicker } from "@web/core/datepicker/datepicker";
+import { Domain } from "@web/core/domain";
+import { Many2XAutocomplete, useOpenMany2XRecord } from "@web/views/fields/relational_utils";
+import { useService } from "@web/core/utils/hooks";
+import { TagsList } from "@web/views/fields/many2many_tags/tags_list";
+import { m2oTupleFromData } from "@web/views/fields/many2one/many2one_field";
+import { PropertyTags } from "./property_tags";
+import { AutoComplete } from "@web/core/autocomplete/autocomplete";
+
+const { Component } = owl;
+const { DateTime } = luxon;
+
+// Formats to stringify the date / datetime in the JSON.
+// It's important to have the year first, then the day,
+// etc... in UTC, to be able to search on them.
+const DATETIME_FORMAT = 'yyyy-LL-dd HH:mm:ss';
+const DATE_FORMAT = 'yyyy-LL-dd';
```

Comment:
So that's based on server datetime format, right?
Should probably call it DEFAULT_SERVER_DATETIME_FORMAT to match the python version then?